README.md

```diff
@@ -24,7 +24,9 @@ Optionally, you may choose to build this repository using Docker, for
 convenience.
 Make sure you have a recent version of [Docker](https://docs.docker.com/) and
 [nvidia-docker](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker)
-installed. Next to get started simply run the following command.
+installed.
+And [rocker](https://github.com/osrf/rocker) available as debian package `python3-rocker` with the ROS packages.
```

Comment:
Nitpick: this project doesn't use ROS, so users landing here may not have ROS packages setup. They may setup it at later stages to get `colcon`, but up here on the README, it may not be clear how to set it up.

I always prefer to err on the side of explicitness in this situation to avoid confusing new users who may not be familiar with ROS packages. So I'd suggest either removing the part about "_debian package `python3-rocker` with the ROS packages_" and letting the `rocker` entry page explain how to set it up, or repeating the step-by-step here on the README.